cloud/passportauth/src/user/BaseUser.ts

```diff
@@ -0,0 +1,32 @@
+export interface BaseUser {
```

Comment:
UserAPI should be interface and BaseUser should be base implementation on UserAPI interface as @paolobueno  mentioned above